lib/active_support/cache/memcached_store.rb

```diff
@@ -71,16 +56,18 @@ def read_multi(*names)
         values = {}
 
         instrument(:read_multi, names, options) do
-          if raw_values = @data.get_multi(keys_to_names.keys, raw: true)
+          if raw_values = @data.get(keys_to_names.keys, false)
```

Comment:
`Memcached::Rails` negated these before passing to `Memcached` super's... 
The former thinks of it as in `raw` while the latter as in `decode` where `raw == !decode` Gymnastic for the mind of the reader ;)